src/client/src/apps/MainRoute/widgets/spotTile/components/Tile/Tile.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import { CurrencyPair, Direction, ServiceConnectionStatus } from 'rt-types'
-import { createTradeRequest, ExecuteTradeRequest, SpotTileData, TradeRequest } from '../../model'
+import { createTradeRequest, ExecuteTradeRequest, SpotTileDataWithNotional, TradeRequest } from '../../model'
```

Comment:
name change since we now have two types - `SpotTileData` and `SpotTileDataWithNotional` (maybe this is a horrible name). Bottom line - `SpotTileData` is in store with optional notional, but there's a selector that always adds `notional` into existing `SpotTileData`. the result is `SpotTileDataWithNotional`. This has been done because we don't know notional when we create `SpotTileData` - default `notional` fill change from currency to currency. Thus we are adding a default `notional` in the selector (we add it based on currency).

I can't say I like this approach - I would prefer to do this in a reducer. But it would have been much bigger change - resetting reference data would need to create new tiles in reducer, then new tiles will need to reset all `tileData` slice. I'd like to have that and left some TODOs there

Reply:
Can you paste this explanation in to the `spotTile.ts` file

Reply:
done